### ISSUE TITLE
May I use Cifar10 to train the generator? Why haven't I been converging? If you can, please provide the code. Thank you.

### ISSUE THREAD

**Issue by tanlingp** [Alibaba-AAIG/Beyond-ImageNet-Attack#8]
May I use Cifar10 to train the generator? Why haven't I been converging? If you can, please provide the code. Thank you.

**Comment by qilong-zhang** [comment]
Hi @tanlingp , we haven't use CIFAR-10 to train our generator. Have you tried to reduce the learning rate during the training phase? I'm guessing the batch size is set relatively large.

**Comment by tanlingp** [comment]
#I am trying to use the CIFAR10 dataset as the source domain to train the generator, but it has been unable to converge. If you can, please provide the code. Thank you.

**Comment by qilong-zhang** [comment]
> #I am trying to use the CIFAR10 dataset as the source domain to train the generator, but it has been unable to converge. If you can, please provide the code. Thank you.

What's the learning rate and batch size in your implementation?

**Comment by tanlingp** [comment]
the learnning rate set 0.00001，batch size set 128



| |
tlp6378
|
|
***@***.***
|




---- Replied Message ----
| From | Qilong ***@***.***> |
| Date | 05/17/2023 10:00 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [Alibaba-AAIG/Beyond-ImageNet-Attack] May I use Cifar10 to train the generator? Why haven't I been converging? If you can, please provide the code. Thank you. (Issue #8) |

#I am trying to use the CIFAR10 dataset as the source domain to train the generator, but it has been unable to converge. If you can, please provide the code. Thank you.

What's the learning rate and batch size in your implementation?

**Comment by qilong-zhang** [comment]
> the learnning rate set 0.00001，batch size set 128 | | tlp6378 | | ***@***.*** |
> […](#)
> ---- Replied Message ---- | From | Qilong ***@***.***> | | Date | 05/17/2023 10:00 | | To | ***@***.***> | | Cc | ***@***.***>***@***.***> | | Subject | Re: [Alibaba-AAIG/Beyond-ImageNet-Attack] May I use Cifar10 to train the generator? Why haven't I been converging? If you can, please provide the code. Thank you. (Issue #8) | #I am trying to use the CIFAR10 dataset as the source domain to train the generator, but it has been unable to converge. If you can, please provide the code. Thank you. What's the learning rate and batch size in your implementation? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

Try to resize CIFAR-10 from 32x32 to 224x224 beforehand. This may be helpful.
